pkg/kubelet/prober/worker.go

```diff
@@ -62,6 +63,11 @@ type worker struct {
 	resultRun int
 }
 
+const (
+	// jitter factor for worker.doProbe sync loop
+	workerProbeJitterFactor = 1.0
```

Comment:
+1 that the factor is too large. 

On the other hand, with a small factor, the effect may be limited. Given that there are some inherent variation as to when the prober starts, do we need this?